modular_skyrat/modules/customization/modules/food_and_drinks/food/misc.dm

```diff
@@ -0,0 +1,21 @@
+//For miscellaneous foods
+
+//cytology mobs butcher results
+/obj/item/food/grown/grapes/ooze
+	seed = /obj/item/seeds/grape/green
+	name = "bunch of oozing grapes"
+	icon_state = "greengrapes"
+	bite_consumption_mod = 3
+	tastes = list("gelatinous grape" = 1)
+	distill_reagent = /datum/reagent/toxin/slimejelly
+
+/obj/item/food/octopusrings
```

Comment:
ewww

Reply:
im not good at balancing so be honest if its too much